app/api/pet.py

```
@@ -1,20 +1,13 @@
-)
+pet_api = APIRouter(tags=["pets"])
 
 # 반려견 생성
-@llm_api.post("/")
+@pet_api.post("/")
 async def create_pet(pet: PetRequest):
-    
-
-    return JSONResponse(status_code=201)
\ No newline at end of file
+    id = create_pet(pet)
```

Ruff (0.12.2)

Local variable `id` is assigned to but never used

Remove assignment to unused variable `id`

(F841)